src/api/app/views/webui/obs_factory/staging_projects/_checks.html.haml

```diff
@@ -1,15 +1,16 @@
+- checks = project.checks
+- missing_checks = project.missing_checks
```

Comment:
Why not setting this in the controller or passing it as local variables to the partial?

Reply:
I can do that